aytests/btrfs_copy_on_write_subvolumes.sh

```diff
@@ -5,13 +5,13 @@
 
 set -e -x
 
-# search for the last '-' in "----------------- /" (copy-on-write enabled by default)
-lsattr -d / | grep -q -- "- /"
+# search for no flags '---' (copy-on-write enabled by default)
```

Comment:
Format has changed. Using `lsattr -dl` looks like a more robust way to check for flags:

```
/var/lib/pgsql               No_COW
```